exporter_test.go

```diff
@@ -102,6 +103,93 @@ func testExporter(t *testing.T, when spec.G, it spec.S) {
 	})
 
 	when("#Export", func() {
+		when("previous slice image exists", func() {
+			var (
+				fakeOriginalImage *fakes.Image
+				fakeImageMetadata lifecycle.LayersMetadata
+				sliceSHA          string
+			)
+
+			it.Before(func() {
+				var err error
+
+				appDir, err = filepath.Abs(filepath.Join("testdata", "exporter", "previous-slice-image-exists", "layers", "app"))
+				h.AssertNil(t, err)
+
+				sliceConfigFilePath := filepath.Join(appDir, "static", "assets", "config.txt")
```

Comment:
Any reason not to make these test fixtures, like the other files in the `appDir`?

Reply:
@ekcasey now that I'm modifying the temp `appDir` (inside the copied temp dir) there maybe opportunities to consolidate the fixture data.

Reply:
That makes sense to me. We could use the same app fixture in both the slices/no-slices cases. Either all of the above files are added to a single layer or they are carved up into slice layers, depending on the values in `metadata.toml`.